### PR TITLE
[TAS-2196] 🚸 Add Crisp help for empty content in /bookshelf

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-09-10 下午1 32 48
截圖 2024-09-10 下午1 35 46

### CODE REVIEW

File: src/components/NFTPortfolio/Empty.vue [hunk 1]
```diff
@@ -13,26 +13,59 @@
...
+    shouldShowCrisp() {
+      return (
+        this.$route.name.includes('bookshelf') && this.preset === 'collected'
+      );
+    },
+  },
+  methods: {
+    handleClickHelp() {
+      logTrackerEvent(this, 'bookshelf', 'bookshelf_crisp_click', '', 1);
+      if (this.$crisp) {
```
Comment (williamchong) [inline src/components/NFTPortfolio/Empty.vue:64]: Any fallback if we dont have crisp?

File: src/components/NFTPortfolio/Empty.vue [hunk 2]
```diff
@@ -13,26 +13,59 @@
...
+        this.showCrisp();
+        this.$crisp.push(['do', 'chat:open']);
```
Comment (williamchong) [inline src/components/NFTPortfolio/Empty.vue:66]: Maybe prefill the content also

File: src/locales/en.json [hunk 3]
```diff
@@ -532,6 +532,9 @@
   "nft_claim_claimed_content_manualDelivery": "Please wait for the author to sign and send the eBook, which may take about 1-3 days. Any update will be notified via email. Thank you for your support!",
   "nft_claim_claimed_content_mixedDelivery": "Some of the book requires author to sign and send the eBook, which may take about 1-3 days. Any update will be notified via email. For auto delivered books, visit your dashboard to start reading. Thank you for your support!",
   "nft_claim_claimed_download": "Other Ways to Obtain the eBook",
+  "nft_claim_claimed_empty_collected_help": "Can't find your purchased content? Seek {help}.",
+  "nft_claim_claimed_empty_crisp_prefilled_message": "Hi, I am unable to find the e-book I recently purchased. Could you please help me locate it? Thank you!",
```
Comment (williamchong) [inline src/locales/en.json:536]:
```suggestion
  "nft_claim_claimed_empty_crisp_prefilled_message": "Hi, I am unable to find the ebook I recently purchased. Could you please help me locate it? Thank you!",

```

File: src/components/NFTPortfolio/Empty.vue [hunk 4]
```diff
@@ -13,26 +13,63 @@
...
+  },
+  methods: {
+    handleClickHelp() {
+      logTrackerEvent(this, 'bookshelf', 'bookshelf_crisp_click', '', 1);
+      const res = this.openCrisp(
+        this.$t('nft_claim_claimed_empty_crisp_prefilled_message')
+      );
+      if (!res) {
+        window.open(
+          'https://discord.com/channels/763001015712350231/814761730349596712'
```
Comment (williamchong) [inline src/components/NFTPortfolio/Empty.vue:69]: Maybe use `https://go.crisp.chat/chat/embed/?website_id=....` , like in about nft book page `publishStoryURL`

File: src/locales/en.json (hunk 3 above)
Comment (AuroraHuang22) [reply src/locales/en.json]: [💬 Update wording for prefilled msg](https://github.com/likecoin/liker-land/pull/1787/commits/5ffdcfd275a1a294e014ab5e45b7d1bea242f191)

File: src/components/NFTPortfolio/Empty.vue (hunk 4 above)
Comment (AuroraHuang22) [reply src/components/NFTPortfolio/Empty.vue]: [✨ Add crisp chat site link as fallback](https://github.com/likecoin/liker-land/pull/1787/commits/3461fc6ebb231c63d7701517085dbb99086fb0b1)

**Review by williamchong — APPROVED** [review]

**Review by nwingt — APPROVED** [review]

**Pull request merged**